adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -0,0 +1,103 @@
+import "Optimus.wdl" as Optimus
+import "submit.wdl" as submit_wdl
+
+
+task GetInputs {
+  String bundle_uuid
+  String bundle_version
+  String dss_url
+  Int retry_seconds
+  Int timeout_seconds
+
+  command <<<
+    python <<CODE
+    from pipeline_tools import utils
```

Comment:
`from pipeline_tools import dcp_utils` if you want to use `humancellatlas/pipeline-tools:0.1.5`